tests/backend/test_speculos.py

```diff
@@ -149,3 +149,41 @@ def test_clicks(self):
                     self.backend.right_click()
                     self.backend.left_click()
                     self.backend.both_click()
+    
+    def test_navigate_until_snap(self):
+        with patch("speculos.client.subprocess"):
+            with SpeculosServerStub():
+                with self.backend:
+                    ret = self.backend.navigate_until_snap(Path(__file__).parent.parent.resolve(),"generic",0,2)
```

Comment:
As it's used several times, you could define a constant with the path prefix on top of the file.
```python
ROOT_SCREENSHOT_PATH = Path(__file__).parent.parent.resolve()
```

Reply:
Done.